eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           AdditionalTestArgs: ${{ parameters.AdditionalTestArgs }}
           TestMarkArgument: ${{ parameters.TestMarkArgument }}
 
-  - job: 'Test_Linux_Python38'
+  - job: '${{ parameters.JobName }}_Linux_Python38'
```

Comment:
Let's add a `dependsOn` for `Test_Linux_PyPy3`. This should force it to run individually.